src/Waypoint.h

```diff
@@ -18,6 +18,7 @@ class Waypoint: public MapObject {
         virtual QString toolTip() const;
         virtual void showDetailsDialog() {} // not applicable
         virtual int type() { return 0;}
+        QString regionCode;
```

Comment:
Wow, that's cool :) I haven't known that we have a region code in the NavData. This will improve Waypoint matching in routes!